### ISSUE TITLE
Using imgproc::find_contours ?

### ISSUE THREAD

**Issue by Tramac** [twistedfall/opencv-rust#416]
```
// gray_image_slice is slice &[u8]
let gray_image = image::GrayImage::from_raw(240, 360, gray_image_slice.to_vec()).unwrap();
// gray_image.save(&"gray_image.png");

let mut contours_cv = VectorOfVectorOfPoint::new();
imgproc::find_contours(
    &gray_image, 
    &mut contours_cv, 
    imgproc::RETR_EXTERNAL,
    imgproc::CHAIN_APPROX_SIMPLE,
    core::Point::new(0, 0),
).unwrap();
```

The following error occurred when compiling：
<img width="804" alt="image" src="https://user-images.githubusercontent.com/22740819/215312107-bbf51016-fe5c-4aa4-8e45-9fc5d996d68d.png">

The `gray_image` is type of `ImageBuffer`, but `imgproc::find_contours` need `ToInputArray`, I have no idea how to convert `ImageBuffer` to `ToInputArray` format, help please.

**Comment by twistedfall** [comment]
The easiest way is to use OpenCV function `imgcodecs::imread` to load image, like in this example: https://github.com/twistedfall/opencv-rust/blob/master/examples/window.rs

**Comment by Tramac** [comment]
Thanks for your reply, but there is no way to load image locally, only slice given.

**Comment by twistedfall** [comment]
If you only have a byte slice then you'd need to manually make a `Mat` and copy the data over into it. I'm planning to make a release later today with the helper function for that. Alternatively you can use the `rel` branch and call the `Mat::from_slice_rows_cols()` function (https://github.com/twistedfall/opencv-rust/blob/rel/src/manual/core/mat.rs#L172). It should look something like this for your case:
```rust
let gray_image = Mat::from_slice_rows_cols(gray_image_slice, 360, 240).unwrap();
```

**Comment by twistedfall** [comment]
This should now be available in 0.76.3 release
